06-GitShare.Rmd

```diff
@@ -23,19 +23,19 @@ knitr::include_graphics("images/GitFork.PNG")
 
 -   Check out a new `branch` using the instructions previously provided in [Use branches to avoid heartache](#Best5.2). This will make a branch of your forked repo.
 
--   You will create a `pull request` when you are finished contributing to the branch. The difference now is that the repo owner will need to `Compare & Pull request` on their Github account. This given them a chance to review your changes before merging them with the main branch.
```

Comment:
The Compare and Pull Request button only shows up if someone has made a recent change. It may be worth giving general instructions, in case people go to do the pull request a day later or so. Once they're more familiar, the button should be self-explanatory when they see it later.